### PR TITLE
Fix Mobile: Add busy in hashtag channel voice

### PR DESCRIPTION
**Pull request by vyhuynhngoc**
Mobile App: display busy in hashtag channel voice
[#2797](https://github.com/nccasia/mezon-fe/issues/2797)

### CODE REVIEW

File: apps/mobile/src/app/components/Suggestions/SuggestItem/index.tsx [hunk 1]
```diff
@@ -11,12 +15,23 @@ type SuggestItemProps = {
...
-const SuggestItem = ({ avatarUrl, symbol, name, subText, isDisplayDefaultAvatar, isRoleUser, emojiId }: SuggestItemProps) => {
+const SuggestItem = ({ channelId, avatarUrl, symbol, name, subText, isDisplayDefaultAvatar, isRoleUser, emojiId }: SuggestItemProps) => {
 	const { themeValue } = useTheme();
 	const styles = style(themeValue);
 	const emojiSrc = emojiId ? getSrcEmoji(emojiId) : '';
+	const voiceChannelMember = useSelector(selectVoiceChannelMembersByChannelId(channelId));
+	const { t } = useTranslation(['clan']);
+
+	const checkVoiceStatus = useMemo(() => {
+		if (channelId !== undefined && voiceChannelMember) {
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/components/Suggestions/SuggestItem/index.tsx]: change `!!channelId` to cover case `null`, `''` instead of just `undefined`

**Comment by phuocnguyenncc** [comment]
@vyhuynhngoc change PR into `develop`

File: apps/mobile/src/app/components/Suggestions/SuggestItem/index.tsx (hunk 1 above)
Comment (vyhuynhngoc) [reply apps/mobile/src/app/components/Suggestions/SuggestItem/index.tsx]: em fix rồi ạ

**Review by phuocnguyenncc — APPROVED** [review]

File: apps/mobile/src/app/components/Suggestions/SuggestItem/index.tsx [hunk 2]
```diff
@@ -11,12 +15,23 @@ type SuggestItemProps = {
...
+	channelId?: string
 };
 
-const SuggestItem = ({ avatarUrl, symbol, name, subText, isDisplayDefaultAvatar, isRoleUser, emojiId }: SuggestItemProps) => {
+const SuggestItem = ({ channelId, avatarUrl, symbol, name, subText, isDisplayDefaultAvatar, isRoleUser, emojiId }: SuggestItemProps) => {
 	const { themeValue } = useTheme();
 	const styles = style(themeValue);
 	const emojiSrc = emojiId ? getSrcEmoji(emojiId) : '';
+	const voiceChannelMember = useSelector(selectVoiceChannelMembersByChannelId(channelId));
+	const { t } = useTranslation(['clan']);
```
Comment (nguyentrannhan) [inline apps/mobile/src/app/components/Suggestions/SuggestItem/index.tsx:23]: Dung chung voi web

Comment (vyhuynhngoc) [reply apps/mobile/src/app/components/Suggestions/SuggestItem/index.tsx:23]: Dạ em sửa rồi á anh.

**Pull request merged**